### DISCUSSION TITLE
Custom Metadata for Error Pages in Next.js 13

### DISCUSSION THREAD

**Discussion by Antonio112009** [vercel/next.js#57785] [Help]
### Summary

I'm exploring ways to enhance the user experience on error pages in Next.js 13. I wonder if it's possible to add hardcoded custom metadata, such as <title> and <meta description>, so that when a user encounters a 404 or 500 error, the browser tab displays a more user-friendly and descriptive message. For example, for a 404 error, the tab could show something like "404: Lost in the Cosmos?".

Here is a simplified example of what the metadata could look like:
```js
//not-found.js
//_______________________
'use client' 

import React from 'react'
import Link from "next/link";


export const metadata = {
    title: '404: Lost in the Cosmos?',
    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolore, labore.'
}

export default function Custom404() {

    return (
       ... Some code
    )
}

### Additional information

_No response_

### Example

_No response_

**Comment by icyJoseph** [comment]
You can add metadata to the `not-found.tsx` pages right? At least to the root one.

And for dynamic paths, I think you can use the `generateMetadata` function, check if the dynamic path is valid (the blog post exists), and return metadata for each case. 

Then in the component itself, when you also check if the dynamic path is valid (blog post exists), you render normally, or you call `notFound()`
